src/wrapper.ts

```diff
@@ -275,6 +275,21 @@ export const NATIVE = {
     SentryCapacitor.clearBreadcrumbs();
   },
 
+
+  /**
+   * Closes the Native Layer SDK
+   */
+  closeNativeSdk(): Promise<void> {
+    if (!this.enableNative || !this.isNativeClientAvailable()) {
+      return Promise.resolve();
+    }
```

Comment:
Let's split this into two if's and add logger to log out the information why close on native was not called.